Gruntfile.js

```diff
@@ -25,6 +26,16 @@ module.exports = function (grunt) {
         grunt.log.writeln("Wrote api data to " + outputFile);
     };
 
+    var apiServer = require("./build/api-gen/dynamic-server.js");
+    function runApiServer() {
+      var done = this.async();
+      apiServer(grunt, "./build/api.json");
+      setTimeout(function(){
+        open("http://localhost:8080");
```

Comment:
Why do you have the same `open` here, when you also have a grunt task that does the same?